.vale.ini

```
@@ -3,11 +3,9 @@ MinAlertLevel = suggestion
 
 [*.md]
 BasedOnStyles = Datadog
-Datadog.headings = YES
 Datadog.Trademarks = NO
 BlockIgnores = (?s) *({{< ?code-block [^>]* >}}.*?{{< ?/ ?code-block >}})
 
 [*.html]
 BasedOnStyles = Datadog
-Datadog.headings = NO
 Datadog.Trademarks = YES
```

We have to explicitly say all the exclusions on the html trademark check or it will fail https://gitlab.ddbuild.io/DataDog/documentation/-/jobs/71941823

Something like this

```suggestion
Datadog.Trademarks = YES
Datadog.headings = NO
Datadog.pronouns = NO
Datadog.oxfordcomma = NO
Datadog.latin = NO
Datadog.gender = NO
Datadog.emdash = NO
Datadog.dashes = NO
Datadog.americanspelling = NO
```

I'm sure that will get annoying fast. Perhaps now we have the vale setup in github we can just have the trademark checks run on markdown anyway instead of the html post check.

Other than that this looks good to me